app/src/main/kotlin/cz/covid19cz/erouska/ui/help/HelpVM.kt

```diff
@@ -13,4 +32,81 @@ class HelpVM @ViewModelInject constructor() : BaseVM() {
     fun openChatBot() {
         publish(HelpCommandEvent(HelpCommandEvent.Command.OPEN_CHATBOT))
     }
-}
\ No newline at end of file
+
+    fun searchQuery(query: String?, tvContent: String) {
+        searchJob?.cancel()
+
+        this.queryData.value = query?.trim() ?: ""
+
+        if (queryData.value.length >= 3) {
+            searchJob = viewModelScope.launch {
+                try {
+                    delay(400)
```

Comment:
Add comment in the code why the delay is needed

Reply:
removed in the end